File: C/Is Odd/isOdd.c
#include <stdio.h>
#include <stdbool.h>

bool isOdd(int number)
{
    /* 
    An odd number is an integer of the form n=2k+1, where k is an integer.
    The congruence n ≡ 1 (mod2) holds for odd n.
    */

    // If the number is not divisible by 2 then return true
    return number % 2 == 1 ? true : false;
}

File: JAVA/Is Odd/IsOdd.java
import java.util.Scanner;

public class IsOdd {

    public static void main(String[] args) {

        System.out.println("isOdd? ");
        System.out.println();

        Scanner consoleInput = new Scanner(System.in);

        System.out.print("Number: ");
        long number = consoleInput.nextLong();
        System.out.println();

        boolean isOdd = true;

        //Odd number checker
        if (number % 2 == 0) {
            isOdd = false;
        }

        if (isOdd) {
            System.out.println("Odd Number");
        } else {
            System.out.println("Not an Odd Number");
        }
        System.out.println();

    }

}